source/read.txt

```diff
@@ -0,0 +1,156 @@
+.. _c-read:
+
+======================
+Read Data from MongoDB
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the C Driver to read data from MongoDB.
+   :keywords: usage examples, query, find, code example
+
+.. TODO: uncomment and add to as pages are built
+.. .. toctree::
+..    :titlesonly:
+..    :maxdepth: 1
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common
+functions you can use to retrieve documents by using the {+driver-short+}.
+
+.. tip::
+
+  To learn more about any of the functions shown on this page, see the link
+  to a relevant guide provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <c-read-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<connection string URI>``, with
+the relevant values for your MongoDB deployment.
+
+.. _c-read-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/sample-read-app.c
+    :language: c
+    :copyable:
+    :linenos:
+    :emphasize-lines: 24-26
```

Comment:
Update emphasized lines. This appears to be highlighting below what is intended.